packages/react-components/src/components/Picker/Picker.tsx

```diff
@@ -21,7 +21,7 @@ export interface IPickerProps {
   placeholder?: string;
   isRequired?: boolean;
   noSearchResultText?: string;
-  disableSearch?: boolean;
+  searchDisabled?: boolean;
```

Comment:
I've decided to name it `searchDisabled` as this is an interface visible to consumers.
WDYT?

Reply:
Fine for now